tests/foreman/api/test_host.py

```diff
@@ -298,10 +298,25 @@ def test_positive_create_with_inherited_params(self):
             location=loc,
             organization=org,
         ).create()
-        self.assertEqual(len(host.all_parameters), 2)
+        # get global parameters
+        glob_param_list = {(param.name, param.value) for param
+                           in entities.CommonParameter().search()}
+        # if there are no global parameters, create one
+        if len(glob_param_list) == 0:
+            param_name = gen_string('alpha')
+            param_global_value = gen_string('numeric')
+            entities.CommonParameter(
+                name=param_name,
+                value=param_global_value
+            ).create()
+            glob_param_list = {(param.name, param.value) for param
+                                in entities.CommonParameter().search()}
```

Comment:
@pondrejk you can still not add new global params if you think this is not necessary (as 6.4 must contain them) but
```
assert len(glob_param_list) > 0
```
is a must (that was the perpose of my comments)